### DISCUSSION TITLE
Remove duty cycle limits

### DISCUSSION THREAD

**Discussion by alustig3** [pyControl/code#63] [Ideas]
Hi,

I think it would be nice to allow finer specification of duty cycle, say any integer between 5 and 95?

Is there reasoning for limiting it to 10, 25, 50 and 75% ?

https://github.com/pyControl/code/blob/5a9b9a20b4be4ee65d5658b25e49bbf4d542b544/pyControl/hardware.py#L401

**Comment by ThomasAkam** [comment]
I agree this would be useful, the limited range of duty cycles currently supported was only for implementation simplicity.  

It would also be useful to be able to specify the pulse duration in ms rather than the duty cycle.

**Comment by alustig3** [comment]
for future reference, this was addressed with https://github.com/pyControl/code/commit/3a60f898ef89402a19dd7a70f1d546edc18535b0

Thanks Thomas!
